install_files/ansible-base/tasks/reboot.yml

```diff
@@ -17,6 +17,15 @@
   tags:
     - reboot
 
+- name: Clear out SSH sessions, to prevent reuse of stale ControlPersist file.
+  local_action:
+    module: file
+    args:
+      path: /home/amnesia/.ansible/cp
```

Comment:
Rather than `/home/amnesia/, let's use `~/`, so that it works in all environments. There's a potential side-effect that Ansible use for other non-SD contexts would be briefly affected, requiring new ControlPersist files to be generated, but that seems acceptable.